### PR TITLE
Update All Apps Single Page Wizard Screens to ModalForms

### PR DESCRIPTION
**Pull request by patil-pratik-87**
Screenshot 2022-03-23 at 10 36 57 AM
Screenshot 2022-03-23 at 10 37 48 AM
Screenshot 2022-03-23 at 10 38 30 AM
Screenshot 2022-03-23 at 10 39 05 AM
Screenshot 2022-03-23 at 10 39 45 AM
Screenshot 2022-03-23 at 10 40 22 AM

### CODE REVIEW

File: src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx [hunk 1]
```diff
@@ -133,102 +120,72 @@ const EditRepositoryPage = () => {
...
-                        wizardContext={wizardContext}
-                        setWizardContext={setWizardContext}
-                        username={username}
-                        customerTier={customerTier}
-                        customerTierBlacklist={customerTierBlacklist}
+        {({ setFieldValue, values }) => {
+          return (
+            <>
+              <div className={classes.validatedFormContainer}>
+                <FormFieldSection step={1} title="SETTINGS">
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx:136]: We shouldn't have the titles here as all caps.

File: src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx [hunk 2]
```diff
@@ -133,102 +120,72 @@ const EditRepositoryPage = () => {
...
+                      <Text variant="body2">Repository URL: </Text>
+                      <Text variant="caption1">{repo.url}</Text>
+                    </div>
+                    {values.repositoryType === RepositoryType.Private && (
+                      <div className={classes.loginDetails}>
+                        <TextField
+                          id="username"
+                          label="Username"
+                          className={classes.textField}
+                          onChange={(value) => setFieldValue({ username: value })}
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx:150]: isn't `setFieldValue` a curried fn? is this actually setting the value? or did you mean to use `setMultipleValues` that we did on the call today?

File: src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx (hunk 1 above)
Comment (patil-pratik-87) [reply src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx:136]: yup. Made the change.

File: src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx (hunk 2 above)
Comment (patil-pratik-87) [reply src/app/plugins/kubernetes/components/app-catalog/repositories/edit-repository-page.tsx:150]: Yeah, it's curried. Thanks for pointing it out :). Fixed it

**Review by jruddell — APPROVED** [review]

**Pull request merged**